doc/tutorials/setup_logical_partition_scenario_on_nuc7i7dnb.rst

```diff
@@ -0,0 +1,267 @@
+      and modules on ubuntu of NUC7i7DNB. 
+
+   .. note::
+
+      If image build machine is the target board (NUC7i7DNB) which loads kernel
+      image, need to use ``cp`` copy command to update kernel image (bzImage)
+      and modules on ubuntu of NUC7i7DNB. 
+
+Update ACRN hypervisor Image
+****************************
+
+#. Before building the ACRN hypervisor, you need to figure out the I/O
+   address of the serial port, and the PCI BDF addresses of the SATA controller
+   and the USB controllers on NUC7i7DNB board. Enter the following command to
+   get the I/O addresses of the serial port. NUC7i7DNB boards support one
+   serial port ttyS0. Need to connect the serial port to the development host,
+   in order to access the ACRN serial console to switch between pre-launched
+   VMs:
+   
+   .. code-block:: none
+
+      $ dmesg | grep ttyS0
+      [    0.000000] console [ttyS0] enabled
+      [    1.562546] 00:01: ttyS0 at I/O 0x3f8 (irq = 4, base_baud = 115200) is
+      a 16550A
+
+   The following command prints detailed information about all PCI buses and
+   devices in the system:
+
+   .. code-block:: none
+
+      $ sudo lspci -vv
+      00:14.0 USB controller: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller (rev 21) (prog-if 30 [XHCI])
+              Subsystem: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller
+      00:17.0 SATA controller: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode] (rev 21) (prog-if 01 [AHCI 1.0])
+              Subsystem: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode]
+      00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection I219-LM (rev 21)
+              Subsystem: Intel Corporation Ethernet Connection I219-LM
+
+   .. note:: 
+
+      Double check PCI devices BDF defined in the ``hypervisor/arch/x86/configs/nuc7i7bnh/pci_devices.h``.
+
+#. Clone the ACRN source code and configure the build options
+
+   Please refer :ref:`getting-started-building` to setup ACRN build environment
+
+   Clone the ACRN source code:
+
+   .. code-block:: none
+
+      $ git clone https://github.com/projectacrn/acrn-hypervisor.git
+      $ cd acrn-hypervisor/hypervisor
+
+   Configure the build options:
+
+   .. code-block:: none
+   
+      $ make defconfig BOARD=nuc7i7bnh
+      $ make menuconfig
+
+   Updates the following configure item:
+
+     *  Set ACRN Scenario as "Logical Partition VMs";
+     *  Set Maximum number of VCPUs per VM as "2";
+     *  Set Maximum number of PCPU as "4";
+     *  Clear/Disable “Enable hypervisor relocation”.
+
+   It is recommended to keep the default values of the other items.
+
+#. Check GRUB boot loader name. 
+
+   .. code-block:: none
+   
+      update-grub -V
+      grub-mkconfig (GRUB) 2.02~beta2-36ubuntu3.18
+
+   In this case, the GRUB loader name shall have ``GRUB`` prefix;
+#. Check or update BDF information of PCI devices of each pre-launched VM;
```

Comment:
What needs updating?

Reply:
PCI devices is checked in the previous step, If default pci devices BDF is configured is wrong, user needs to update it according to the results of the previous pci devices BDF checking.